hnn_core/gui/_viz_manager.py

```diff
@@ -149,19 +159,19 @@ def _update_ax(fig, ax, single_simulation, plot_type, plot_config):
 
     elif plot_type == 'network':
         if net_copied:
-            _fig = plt.figure()
-            _ax = _fig.add_subplot(111, projection='3d')
-            net_copied.plot_cells(ax=_ax)
-
-            io_buf = io.BytesIO()
-            _fig.savefig(io_buf, format='raw')
-            io_buf.seek(0)
-            img_arr = np.reshape(np.frombuffer(io_buf.getvalue(),
-                                               dtype=np.uint8),
-                                 newshape=(int(_fig.bbox.bounds[3]),
-                                           int(_fig.bbox.bounds[2]), -1))
-            io_buf.close()
-            ax.imshow(img_arr)
+            with plt.ioff():
```

Comment:
why was this necessary?

Reply:
yes otherwise there will be another figure of the network object under the config tabs like this one:

![image](https://user-images.githubusercontent.com/11160442/190051682-c503b92f-31fe-4912-b846-c8a412433c0b.png)